src/Modal.soy

```diff
@@ -17,7 +18,7 @@
 		class="modal{$elementClasses ? ' ' + $elementClasses : ''}"
 		style="display: {$visible ? 'block' : 'none'}">
 		<div
-			class="modal-dialog"
+			class="modal-dialog{$dialogClasses ? ' ' + $dialogClasses : ''}"
```

Comment:
Could `dialogClasses` attribute have a default empty string to avoid this checking?